### ISSUE TITLE
Behind qt and cocoa version

### ISSUE THREAD

**Issue by Roy-Orbison** [SamSchott/maestral-gui#1]
Running `pip install --upgrade maestral-gui` does nothing, though program itself prompts to upgrade.

**Comment by samschott** [comment]
True. maestral-gui is currently just a placeholder package for a future unified GUI based on [toga](https://github.com/beeware/toga). I had completely forgotten that it actually *does* install the GUI, but it won’t update it to the latest version.

For the time being, please use `pip install -U maestral[gui]` to update.
